### PR TITLE
Update nuspec files to include packages added by core.dll

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

**Comment by bgavrilMS** [comment]
@henrik-me, @jennyf19 - everytime we add a package reference to Core, we need to copy it to the ADAL and MSAL nuspec files, because we are doing packaging manually. 

@henrik-me - we might want to reconsider publishing a package with core.dll for the next version, it would save quite a few manual steps.

File: adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec [hunk 1]
```diff
@@ -32,15 +32,17 @@ ADAL.NET takes advantage of Windows Server Active Directory and Azure Active Dir
...
       <group targetFramework="MonoAndroid8.1">
+        <dependency id="Xamarin.Android.Support.CustomTabs" version="27.0.2" exclude="Build,Analyzers" />
+        <dependency id="Xamarin.Android.Support.v7.AppCompat" version="27.0.2" exclude="Build,Analyzers" />
       </group>
       <group targetFramework="UAP10.0">
         <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.0.8" exclude="Build,Analyzers" />
-        <dependency id="System.Runtime.Serialization.Json" version="4.0.3" exclude="Build,Analyzers" />
-        <dependency id="System.Runtime.Serialization.Primitives" version="4.1.1" exclude="Build,Analyzers" />
-        <dependency id="System.Xml.XDocument" version="4.0.11" exclude="Build,Analyzers" />
+        <dependency id="System.Runtime.Serialization.Json" version="4.3.0" exclude="Build,Analyzers" />
```
Comment (bgavrilMS) [inline adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec:43]: Somewhat unrelated fix, as I was getting NuGet warnings about these referneces. Updated UAP references to 4.3.0 #Resolved

File: adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec [hunk 2]
```diff
@@ -32,15 +32,17 @@ ADAL.NET takes advantage of Windows Server Active Directory and Azure Active Dir
         <dependency id="System.Runtime.Serialization.Json" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Runtime.Serialization.Primitives" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Security.Cryptography.X509Certificates" version="4.3.0" exclude="Build,Analyzers" />
-        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />
+        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />        
       </group>
       <group targetFramework="MonoAndroid8.1">
```
Comment (bgavrilMS) [inline adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec:37]: This is the actual fix - since we bundle core.dll, we need to copy it's dependencies.

Comment (henrik-me) [reply adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec:37]: thanks, great info. perhaps we can add a utest or similar to ensure this is done?

---
In reply to: [210323122](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1201#discussion_r210323122) [](ancestors = 210323122)

**Comment by henrik-me** [comment]
IMO, let's get this change into the release (it was kind of what I was asking about yesterday :) ). At a minimum, I would like this finding to be documented in the RCL. Ideally a utest or similar should catch it ensuring we remember doing this at the right time (I'm happy about other ideas than utests). Aside: the utest can wait until we have shipped. The RCL should be done as part of this work.

---
In reply to: [413248377](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1201#issuecomment-413248377) [](ancestors = 413248377)

**Review by henrik-me — APPROVED** [review]
:shipit:

File: adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec (hunk 1 above)
Comment (jennyf19) [reply adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec:43]: great! thanks.

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Comment by jmprieur** [comment]
what is a RCL, @henrik-me ?

**Comment by bgavrilMS** [comment]
@jmprieur - RCL = Release CheckList, we have some 6 tests that are non-functional like "verify everything is signed" etc.
